qa/workunits/cephtool/test.sh

```diff
@@ -2011,6 +2011,46 @@ function test_mon_cephdf_commands()
   expect_false test $cal_raw_used_size != $raw_used_size
 }
 
+function test_mon_tell_help_command()
+{
+  ceph tell mon.1 help
+
+  # wrong target
+  expect_false ceph tell mon.100 help
+}
+
+function test_osd_tell_help_command()
+{
+  ceph tell osd.1 help
+  expect_false ceph tell osd.100 help
+}
+
+function test_mds_tell_help_command()
+{
+  FS_NAME=cephfs
+  if ! mds_exists ; then
+      echo "Skipping test, no MDS found"
+      return
+  fi
+  remove_all_fs
+  ceph osd pool create fs_data 10
+  ceph osd pool create fs_metadata 10
+  ceph fs new $FS_NAME fs_metadata fs_data
+  wait_mds_active $FS_NAME
```

Comment:
hi, @tchaikov , I test it today. result shows that we need these pools. here is error log.

```
/home/jenkins-build/build/workspace/ceph-pull-requests/qa/workunits/cephtool/test.sh:2035: test_mds_tell_help_command:  ceph tell mds.a help
2017-05-18 06:49:38.874232 7efd58dd3700 -1 WARNING: all dangerous and experimental features are enabled.
2017-05-18 06:49:38.887392 7efd58dd3700 -1 WARNING: all dangerous and experimental features are enabled.
2017-05-18 06:49:38.922283 7efd6706e700 -1 WARNING: all dangerous and experimental features are enabled.
2017-05-18 06:49:38.922596 7efd6706e700 -1 asok(0x23b87d0) AdminSocketConfigObs::init: failed: AdminSocket::bind_and_listen: failed to bind the UNIX domain socket to '/home/jenkins-build/build/workspace/ceph-pull-requests/build/src/test/td/t-7200/out/client.admin.24768.asok': (17) File exists
2017-05-18 06:49:38.927826 7efd6706e700 -1 client.4313 MDS ID 'a' not found
2017-05-18 06:49:38.927829 7efd6706e700 -1 client.4313 FSMap: e91:
couldn't get command descriptions for [u'mds', u'a']: 
/home/jenkins-build/build/workspace/ceph-pull-requests/qa/workunits/cephtool/test.sh:1: test_mds_tell_help_command:  rm -fr /tmp/cephtool.yN1
```